python/shipDet_conf.py

```diff
@@ -91,21 +91,15 @@ def posEcal(z, efile):
     return ecal, EcalZSize
 
 def configure_snd_old(yaml_file,
-                      snd_tauMu_zTot,
-                      snd_tauMu_zMudetC,
+                      Target_zend,
```

Comment:
Please give this a better name. Most people won't think of the SND first when something is called `Target_*`